app/src/main/kotlin/com/intive/picover/common/platform/viewmodel/LatestFirebaseStorageState.kt

```diff
@@ -0,0 +1,10 @@
+package com.intive.picover.common.platform.viewmodel
+
+import android.net.Uri
+
+sealed class LatestFirebaseStorageState {
```

Comment:
Why `LatestFirebaseStorageState` instead of just `FirebaseStorageState`? I would also suggest using feature/screen related names for the state because if I'm not wrong (some MVI expert – correct me 😄) there will be only one state class for the whole screen and the whole screen is not only Firebase Storage.